edx_event_bus_kafka/internal/consumer.py

```diff
@@ -159,8 +159,12 @@ def consume_indefinitely(self):
                     # to commit all this consumer's current offset across all partitions since we only process one
                     # message at a time, but limit it to just the offset/partition of the specified message
                     # to be super safe
-                    self.consumer.commit(message=msg, asynchronous=False)
+                    #
+                    # Use asynchronous commits for performance; if we used synchronous commits we could
+                    # check the return value for errors, but there's really nothing for us to do with that.
+                    self.consumer.commit(message=msg, asynchronous=True)
         finally:
+            # Just notifies the broker that this consumer is shutting down. We have auto-commit disabled.
```

Comment:
I think I’d just leave out this comment. Not sure it adds much, and it theoretically could get out of sync.